recipe/meta.yaml

```diff
@@ -59,7 +59,7 @@ outputs:
         - wheel
```

Comment:
[This skip](https://github.com/AnacondaRecipes/cachecontrol-feedstock/pull/4/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR53) can be updated as well